authors/apps/authentication/tests/test_email_verification.py

```diff
@@ -0,0 +1,44 @@
+from django.urls import reverse
+from rest_framework.test import APIClient
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+
+class TestEmailVerify(APITestCase):
+    '''class for testing email verification'''
+
+    def setUp(self):
+        """
+        Prepare test environment for each testcase
+        """
+        self.client = APIClient()
+        self.signup_url = reverse('authentication:registration')
+        self.user_details = {
+            'user': {
+                'username': 'user1',
+                'email': 'evajohnson714@gmail.com',
+                'password': 'somepass12345',
+            }
+        }
+
+    def test_account_verification(self):
+        """
+        Test whether account was verified in success.
+        """
+        self.client.post(self.signup_url, self.user_details, format='json')
+        response = self.client.get(
+            reverse(
+                "authentication:verify",
+                args=['sometoken']))
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```

Comment:
Should the status be 400?

Reply:
Also test the message returned is correct.

Reply:
Noted, it should be 401

Reply:
we will do that after now. we could not test for that because, there was no actual token to verify agains

Reply:
we should also have a test for a bad json object that will return a status code of 400